awx/settings/defaults.py

```diff
@@ -1067,3 +1062,12 @@
 # Host metric summary monthly task - last time of run
 HOST_METRIC_SUMMARY_TASK_LAST_TS = None
 HOST_METRIC_SUMMARY_TASK_INTERVAL = 7  # days
+
+
+# django-ansible-base
+ANSIBLE_BASE_FEATURES = {'AUTHENTICATION': False, 'SWAGGER': False, 'FILTERING': True}
+
+from ansible_base import settings  # noqa: E402
```

Comment:
What I really don't want here is for the `settings` python object to get included in `DEFAULTS_SNAPSHOT` which is really toxic tech debt, and triggers deepcopy calls, etc... You're treading on thin ice in this file.

My understanding is that this is to include all default settings from ansible_base? Is that correct? Can you add a comment to that effect?

Then, after you do your logic here, I would request

```python
del foo, bar
```

For the reason that logic later on _will inspect locals_, and yes this is horrifying. We need to be super duper deliberate about what variables are being set in this space.